dcpy/utils/versions.py

```diff
@@ -89,6 +89,38 @@ def bump(
         return Quarter(year=new_year, quarter=new_quarter)
 
 
+@dataclass(order=True)
+class Date(Version):
+    year: int
+    month: int
+    day: int
+
+    @property
+    def label(self) -> str:
+        return f"{self.year}-{self.month:02d}-{self.day:02d}"
+
+    def bump(
+        self, version_subtype: VersionSubType | None = None, bump_by: int | None = None
+    ) -> Date:
```

Comment:
Should we just return a NotImplemented error? And leave this unimplemented. I don't think we currently bump anything that would take this format, and maybe we don't want to out of ambiguity (right now only bumps by days rather than months, and I can't think of a case where we'd want to bump the number of days programmatically)